### PR TITLE
ref(test): Convert StackTraceLinkModal to RTL + TS - [TET-360]

### PR DESCRIPTION
**Pull request by priscilawebdev**

### CODE REVIEW

File: fixtures/js-stubs/types.tsx [hunk 1]
```diff
@@ -95,6 +95,12 @@ type TestStubFixtures = {
   PullRequest: OverridableStub;
   Release: (params?: any, healthParams?: any) => any;
   Repository: OverridableStub;
+  RepositoryProjectPathConfig: (
+    project: any,
```
[the remark this answers is not in the document]
Comment (evanpurkhiser) [reply fixtures/js-stubs/types.tsx:99]: Yeah I think it should be fine

[the remark this answers is not in the document]
Comment (evanpurkhiser) [reply fixtures/js-stubs/types.tsx:99]: Should the parameters be part of the `OverridableStub` template parameter?

[the remark this answers is not in the document]
Comment (evanpurkhiser) [reply fixtures/js-stubs/types.tsx:99]: Oh is this new or we just didn't have it in here?

[the remark this answers is not in the document]
Comment (evanpurkhiser) [reply fixtures/js-stubs/types.tsx:99]: I would encourage the you to have the parameters be similar to all the other stubs, where it takes an object not positional arguments.

This way you can use the `OverridableStub` type for it.

[the remark this answers is not in the document]
Comment (priscilawebdev) [reply fixtures/js-stubs/types.tsx:99]: ok 😀 I will have to update other files too

[the remark this answers is not in the document]
Comment (priscilawebdev) [reply fixtures/js-stubs/types.tsx]: done!

File: fixtures/js-stubs/types.tsx [hunk 2]
```diff
@@ -95,6 +95,7 @@ type TestStubFixtures = {
   PullRequest: OverridableStub;
   Release: (params?: any, healthParams?: any) => any;
   Repository: OverridableStub;
+  RepositoryProjectPathConfig: OverridableStub;
```
Comment (evanpurkhiser) [inline fixtures/js-stubs/types.tsx:98]: you can add the types as the template parameter for `OverridableStub` as well, I just didn't when I translated these all over

**Review by evanpurkhiser — APPROVED** [review]

File: fixtures/js-stubs/types.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply fixtures/js-stubs/types.tsx]: the only problem with this approach is that the dev wouldn't know what to pass to the function as the params is of type any

File: fixtures/js-stubs/types.tsx (hunk 2 above)
Comment (priscilawebdev) [reply fixtures/js-stubs/types.tsx:98]: I can do it in a follow-up

**Pull request merged**